src/cpp/session/modules/SessionThemes.R

```diff
@@ -0,0 +1,977 @@
+#
+# SessionThemes.R
+#
+# Copyright (C) 2018 by RStudio, Inc.
+#
+# Unless you have received this program directly from RStudio pursuant
+# to the terms of a commercial license agreement with RStudio, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+
+# ACE/tools/tmThemes.js Functions ==================================================================
+
+# Converts a colour to an array of the RGB values of the colour.
```

Comment:
Canadian spelling of colour in comments will be very controversial. Not really, but doesn't match the spelling in the code itself so probably best to stick to the spelling of our American Overlords.

Reply:
:laughing: